docs/help.md

```diff
@@ -1,7 +1,7 @@
 ### Commands
 `bot adapter` - Reply with the adapter
 
-`bot create|close milestone all "title"` - Create or close all milestones listed in `config.json`
+`bot create|close milestone all "title"` - Creates milestone for **all repos** listed in `config.json`
```

Comment:
Why making it bold?

Reply:
@zelima in your comment, it was bold, or you just showed me?